homework/03LessonThree.js

```diff
@@ -39,9 +40,15 @@ function returnTheFirstArgument(firstArgument) {
 // Write your answer in comments
 //
 // What is scope?
+
 //
 // Your Answer Goes Here:
 //
+// Scope refers to the degree of accessibility to which
+// particular variables, expressions or statements can be
+// be called. The position in the document any of said variables, expressions or
+// statements are defined affects where they can be called.
+//
 //
```

Comment:
Very good! Things that are in the same "scope" would be between the same two curly braces {  }

Here is a very interesting read on scope:

http://javascriptissexy.com/javascript-variable-scope-and-hoisting-explained/